### PR TITLE
Implement Desktop Version of Portfolio Website

### PR DESCRIPTION
**Pull request by CVILLA09**
This pull request includes changes to make the Portfolio website compatible with desktop screens using media queries. The changes were made based on the selected Figma design template for the desktop version.

Key Changes:
- Implemented media queries to make the website responsive for desktop screens (>= 768px).
- Utilized Grid and Flexbox for element layout.
- Ensured the design details (font, colors, images, text, margins) match the Figma template.

General Requirements Met:
- No linter errors.
- Followed the correct Github flow.
- Documented the work in a professional manner.

HTML/CSS Best Practices:
- Followed the best practices for HTML & CSS.

Please review these changes and provide your valuable feedback.

### CODE REVIEW

**Comment by CVILLA09** [comment]
<h1>Hello reviewer! 👋</h1>

<p>I have submitted a new pull request for the review of my portfolio's desktop version. I would like to bring your attention to an issue that I encountered in the desktop version of the section "About Myself". Specifically, there is an issue with the bottom border line in the "Frameworks and Skills" area, as it does not extend along as intended in the Figma design. Despite my attempts to resolve this issue, I have been unable to find a successful solution thus far.

I kindly request your feedback and guidance on how to address this issue effectively. Any assistance you can provide would be greatly appreciated!🙏</p>

GIF

File: style.css [hunk 1]
```diff
@@ -478,3 +483,250 @@ form {
...
+    margin-right: 0; /* Remove the right margin */
+    padding-top: 1%;
+  }
+
+  .skills-section {
+    padding: 0;
+    position: relative;
+    width: 100%;
+    margin-left: 11vw;
+  }
```
Comment (shella12) [inline style.css]: - [ ] Please extend the line under the about me tech categories section to full width to match closely with the provided figma design as according to the requirements you should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


|Figma design|Your design|
|---|----|
|Screenshot 2023-06-16 at 11 04 53 AM|Screenshot 2023-06-16 at 11 21 43 AM|

Suggestion:

Use `min-width:fit-content` and  `max-width:60%` on `.skills-section` to solve the issue. Example snippet:

```css
 .skills-section {
    padding: 0;
    position: relative;
    width: 100%;
    margin-left: 11vw;
    min-width: fit-content;
    max-width: 60%;
  }

```

File: index.html [hunk 2]
```diff
@@ -83,39 +84,41 @@ <h3>CANOPY</h3>
...
-        <p class="project-description">A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+        <p class="project-description">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
         <ul class="project-tags">
           <li><a href="#">HTML</a></li> 
+          <li class="hide-on-mobile"><a href="#">Ruby on rails</a></li> 
           <li><a href="#">CSS</a></li> 
           <li><a href="#">JavaScript</a></li> 
         </ul>
         <button type="button" class="project-btn">See Project</button>
       </div>
```
Comment (shella12) [inline index.html:104]: - [ ] Please add `hover` effects on the `see project` buttons on your project cards as suggested in the figma design

Screenshot 2023-06-16 at 11 05 22 AM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @CVILLA09, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- great design implementation ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (CVILLA09) [reply index.html:104]: <h1>Thanks! You're the best!</h1>

**Comment by CVILLA09** [comment]
<h1>Hello reviewer! 👋</h1>

I'm thrilled to share that I have successfully resolved the following issues: 🎉

1️⃣ I have extended the line under the "About Me" tech categories section to match the full width, closely resembling the provided Figma design. This adjustment aligns with the project requirements, emphasizing the importance of sticking to the design as closely as possible, including font, colors, images, text, and margins. 

2️⃣ I have also added hover effects to the "See Project" buttons on the project cards, just as suggested in the Figma design. Now they truly stand out and catch the user's attention!

I'd like to express my gratitude to the incredible @shella12 for the assistance and guidance in helping me implement these updates. Your expertise has been instrumental in making these improvements. 🙌🙏

**Review by wayungi — APPROVED** [review]
Hi @CVILLA09 , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Good commit messages ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Project up and running ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CVILLA09** [comment]
<h1>Thanks @wayungi! 😊🚀😊</h1>

**Pull request merged**